lib/metrics/gc.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+const Counter = require('../counter');
+const Histogram = require('../histogram');
+
+let perf_hooks;
+
+try {
+	// eslint-disable-next-line
+	perf_hooks = require('perf_hooks');
+} catch (e) {
+	// node version is too old
+}
+
+const NODEJS_GC_RUNS_TOTAL = 'nodejs_gc_runs_total';
+const NODEJS_GC_DURATION = 'nodejs_gc_duration';
+const DEFAULT_GC_DURATION_BUCKETS = [
```

Comment:
I'm having troulbe finding it , but I think recommended label cardinality is `<= 10`. GC durations should be low, and anything approaching 1 second is likely a severe problem. It'd be nice to have a reasonable range, if you have enough production experience to know what that is. I suggest that the cardinality be brought down to 7, these aren't supposed to be very fine-grained, maybe?
```
[0.001, 0.01, 0.1, 1, 2, 5]
```
Or pehaps no 5, but one more bucket at the low end?

Reply:
I think what is "reasonable range" really depends on what your software is doing.  I think [0.001, 0.01, 0.1, 1, 2, 5] buckets can be a good point to start with. Updated defaults